docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to proxy peering mode.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  TLS routing is available starting from Teleport `10.0`.
+</Details>
+
+## Requirements
+
+Teleport Proxy servers must be able to reach each other over the network. Ensure
```

Comment:
```suggestion
Teleport Proxy Service instances must be able to reach each other over the network. Ensure
```

Reply:
Replaced with just `instances` rather than `Service instances` for consistency.